publication/models/account_analytic_invoice_line.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2019 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# pylint: disable=no-member,too-few-public-methods
```

Comment:
To be removed?

Reply:
These are there to prevent spurious linting errors. pylint does not understand the Odoo inheritance mechanism, therefore does not see the members and methods inherited from other models.